### PR TITLE
Update Red Eyes tutorial to Alderaan

### LINKED ISSUE #5431: Extend the Mainnet Tutorial in the docs for Alderaan
For the docs of the Alderaan release, we want an actionable step-to-step guide for new users that explains
- how to install and run Raiden from the command line and
- how to initiate transfers using the Raiden API. 

The existing “Installation guide” and “Red Eyes mainnet tutorial” sections in the docs will be rewritten accordingly.

- [x] Update the tutorial to Alderaan (changed API in examples, whitelisted tokens, etc)

- [x] Update the installation guide if necessary

- [x] Add use of PFS to the tutorial (since we now suggest against internal routing)

- [x] Explain how to obtain and collateralize Raiden tokens for the services

- [x] Add information about the monitoring service to the tutorial too

- [x] Rewrite both sections for readability/step-by-step structure

### PR DESCRIPTION
**Pull request by jomuel**
Update the Red Eyes tutorial in the docs, and the installation guide it relies on, to Alderaan. The role of the services has changed, a new token will be whitelisted, and one of the example requests was outdated too.

Closes #5431

### CODE REVIEW

File: docs/overview_and_guide.rst [hunk 1]
```diff
@@ -227,20 +237,84 @@ After account creation, launch Raiden with the path of your keystore supplied::
...
+where ``network`` is one of ``mainnet``, ``ropsten``, ``rinkeby`` or ``goerli``.
+
+Log into MyEtherWallet, the geth console or any other Ethereum interface you like with an account
+that holds enough Raiden tokens, and call ``UserDeposit.deposit(<beneficiary>, <total_deposit>)``.
+
+Here ``total_deposit`` is the amount of Raiden tokens you wish to deposit and ``beneficiary`` is the
+Ethereum address of your Raiden node.
+
+For example, suppose we use Raiden with the address ``0x3040435D7F1012e861f0B0989422a47D1825F120`` on the mainnet and want
+to deposit 0.001 Raiden tokens (10**15 REI) for our Raiden node to use. We take a look at ``deployment_services_mainnet.json``, which yields:
```
Comment (Dominik1999) [inline docs/overview_and_guide.rst]: Can you change `0.001 Raiden tokens` to `6 Raiden tokens`? Then the user can get at least one channel monitored.

File: docs/overview_and_guide.rst [hunk 2]
```diff
@@ -227,20 +237,84 @@ After account creation, launch Raiden with the path of your keystore supplied::
...
+This is discouraged since the node has less information about the network that it can use to find routes and compute fees.
+So transfers will be more likely to fail to route, and paid fees will often be unnecessarily high when internal routing is used.
+If you want to use internal routing anyway, you can do so with ``--routing-mode local``.
+
+.. note::
+    Although otherwise discouraged, ``--routing-mode local`` has to be used at the moment to try
+    out Raiden on the mainnet. The Raiden Service Bundle, with pathfinding service and registry,
+    will not be deployed on the mainnet until the Alderaan release.
+
+If you want to use a particular pathfinding service, e. g. one of the testnet pathfinding services given below, you can
```
Comment (Dominik1999) [inline docs/overview_and_guide.rst]: There is one space too much between `e.` and `g.`, right?

**Comment by Dominik1999** [comment]
and are you sure someone, who is not familiar with Raiden yet, can execute a mediated transfer in W-ETH with the help of this tutorial?

To me it looks like 

``` 
Let's say we know someone with the address ``0x00014853D700AE1F39BA9dbAbdeC1c8683CF1b2A``, who is also part of the W-ETH token network. Even though we do not have a channel with this person it is as easy as above to send a payment. All we need is the address:
.. http:example:: curl wget httpie python-requests
    POST /api/v1/payments/0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2/0x00014853D700AE1F39BA9dbAbdeC1c8683CF1b2A HTTP/1.1
    Host: localhost:5001
    Content-Type: application/json
    {
        "amount": "73"
    }
Just like this we can send payments to anyone who is part of the token network for the W-ETH token.
``` 

this is not sufficient, right? Still the person can not use a pathfinding service, right? Or am I missing something here?

File: docs/overview_and_guide.rst (hunk 1 above)
Comment (jomuel) [reply docs/overview_and_guide.rst]: Good point. I'll double-check if the parameter is really in REI not RDN too.

File: docs/alderaan_mainnet_tutorial.rst [hunk 3]
```diff
@@ -99,7 +105,10 @@ Now that we have joined a token network, we can start making payments. For this,
         "amount": "42"
     }
 
-In this specific case the payment goes directly to one of our channel partners, since we have an open channel with ``0x61C808D82A3Ac53231750daDc13c777b59310bD9``. If we specify an address that we do not have a direct channel with, the Raiden Network finds a path to the target address and use mediated transfers to make a payment from our address to the target address.
+This is an example of a direct transfer. Since we have an channel with the address we are sending to, ``0x61C808D82A3Ac53231750daDc13c777b59310bD9``, the payment goes straight to it. If we specify an address that we do not have a direct channel with, Raiden will try to do a mediated transfer, i. e.
+to find a path from us to the target address in the network of channels.
+
+ with the help of either its internal routing or the given pathfinding service.
```
[the remark this answers is not in the document]
Comment (jomuel) [reply docs/alderaan_mainnet_tutorial.rst]: I think this line is just a leftover to be removed

File: docs/overview_and_guide.rst (hunk 2 above)
Comment (jomuel) [reply docs/overview_and_guide.rst]: Really? I wasn't sure if its spelled with a whitespace

File: docs/alderaan_mainnet_tutorial.rst [hunk 4]
```diff
@@ -58,8 +64,8 @@ In case we know of a specific address in the network that we will do frequent pa
    {
        "partner_address": "0x61C808D82A3Ac53231750daDc13c777b59310bD9",
        "token_address": "0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2",
-       "balance": "2000",
-       "settle_timeout": "500"
+       "total_deposit": 2000,
+       "settle_timeout": 500
    }
 
 At this point the specific value of the ``balance`` field isn't too important, since it's always possible to :ref:`deposit more tokens <topping-up-a-channel>` to a channel if need be.
```
[the remark this answers is not in the document]
Comment (jomuel) [reply docs/alderaan_mainnet_tutorial.rst]: Yes, it should be `total_deposit`

**Comment by jomuel** [comment]
> this is not sufficient, right? Still the person can not use a pathfinding service, right? Or am I missing something here?

Why not, what information is missing about the pathfinding service? We have explained what the service registry does, how the raiden tokens are deposited and how the routing mode is set above.

**Comment by Dominik1999** [comment]
> > this is not sufficient, right? Still the person can not use a pathfinding service, right? Or am I missing something here?
> 
> Why not, what information is missing about the pathfinding service? We have explained what the service registry does, how the raiden tokens are deposited and how the routing mode is set above.

Can we push this PR then let's try it if we can simply execute it and do a mediated transfer?

File: docs/overview_and_guide.rst [hunk 5]
```diff
@@ -227,20 +237,84 @@ After account creation, launch Raiden with the path of your keystore supplied::
...
+
+All services that are registered in the service registry of a given network will use one shared instance of ``UserDeposit``.
+You can obtain the address of the contract from
+
+    ``https://github.com/raiden-network/raiden-contracts/blob/master/raiden_contracts/data/deployment_services_<network>.json``
+
+where ``network`` is one of ``mainnet``, ``ropsten``, ``rinkeby`` or ``goerli``.
+
+Log into MyEtherWallet, the geth console or any other Ethereum interface you like with an account
+that holds enough Raiden tokens, and call ``UserDeposit.deposit(<beneficiary>, <total_deposit>)``.
```
Comment (palango) [inline docs/overview_and_guide.rst]: Don't we need to call `approve` on the token here before calling `deposit`? @jomuel

**Pull request merged**